packages/cli/src/explore/utils.ts

```diff
@@ -12,3 +13,22 @@ export function shortDate(date?: string): string {
   if (!date) return ''
   return formatDistanceToNow(new Date(date))
 }
+
+// return true if copy worked, false otherwise
+export function copyToClipboard(text: string): boolean {
+  /**
+   * There's probably a better way than spawning. We tried and failed at the following:
+   * - screen.copyToClipboard didn't do anything (and is iTerm2-specific, anyway)
+   * - clipboardy wouldn't run due to error "Must use import to load ES Module" (even though we used import)
+   */
+  if (os.platform() === 'darwin') {
```

Comment:
one thing I noticed after my review, this doesn't seem to support linux. imo, it's fine to merge it as is, as long as we create an issue to add linux support (could be a "good first issue")

Reply:
Good call: [issue logged here](https://github.com/uport-project/veramo/issues/973).